kinogoteborg/src/app/components/headerFooter/footer.jsx

```diff
@@ -5,18 +5,12 @@ export default function Footer() {
   return (
     <div className="fixed bottom-0 bg-[#800020] w-full pt-10 flex justify-center">
```

Comment:
"fixed" messes with the admin page.
this is howerver changed to "relative" in renderSaloon PR